client/asset/btc/regnet_test.go

```diff
@@ -332,6 +334,12 @@ func TestWallet(t *testing.T) {
 		t.Fatalf("refund error: %v", err)
 	}
 
+	// // Test PayFee
+	// _, err = rig.gamma().PayFee(alphaAddress, 1e8, tBTC)
+	// if err != nil {
+	// 	t.Fatalf("error paying fees: %v", err)
+	// }
```

Comment:
Will you be removing this?

Reply:
You must be looking at an older commit. It's already fixed.

Reply:
yeah was reviewing the commits top down, just noticed.